apps/api/src/app/integrations/integrations.controller.ts

```diff
@@ -60,7 +60,7 @@ export class IntegrationsController {
 
   @Get('/')
   @ApiOkResponse({
-    type: IntegrationResponseDto,
+    type: IntegrationResponseDto[],
```

Comment:
I'm not sure if this will solve the issue, I believe there is an `isArray` property that needs to be passed, and this should be as it was previously.

Reply:
I have made changes based on what was used in other places, should work now